<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QDockWidget" name="dockWidget">
     <widget class="QWidget" name="dockWidgetContents">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>398</width>
            <height>703</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="0">
            <widget class="QPushButton" name="logsFolder">
             <property name="text">
              <string>Select Logs Folder</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QLineEdit" name="init_embedding_manager_ckpt"/>
           </item>
           <item row="6" column="0">
            <widget class="QPushButton" name="initEmbeddingManager">
             <property name="text">
              <string>Init embedding manager from checkpoint</string>
             </property>
            </widget>
           </item>
           <item row="14" column="1">
            <widget class="QCheckBox" name="no_test">
             <property name="text">
              <string>No Test</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="19" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Steps tuned</string>
             </property>
            </widget>
           </item>
           <item row="15" column="0" colspan="2">
            <widget class="QPushButton" name="resumeModel">
             <property name="text">
              <string>Model to resume from</string>
             </property>
            </widget>
           </item>
           <item row="20" column="1">
            <widget class="QLCDNumber" name="steps_tuned"/>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="pathInputImages">
             <property name="text">
              <string>Select path to input Images</string>
             </property>
            </widget>
           </item>
           <item row="22" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Placeholder Tokens</string>
             </property>
            </widget>
           </item>
           <item row="21" column="1">
            <widget class="QPushButton" name="Stop">
             <property name="text">
              <string>Stop finetuning</string>
             </property>
            </widget>
           </item>
           <item row="16" column="0" colspan="2">
            <widget class="QLineEdit" name="actual_resume">
             <property name="text">
              <string>data/models/sd-v1-4.ckpt</string>
             </property>
            </widget>
           </item>
           <item row="13" column="0" colspan="2">
            <widget class="QLineEdit" name="init_word"/>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="name">
             <property name="text">
              <string>TestProject</string>
             </property>
            </widget>
           </item>
           <item row="21" column="0">
            <widget class="QPushButton" name="Start">
             <property name="text">
              <string>Start finetuning</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <widget class="QCheckBox" name="train">
             <property name="text">
              <string>Train</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Output Name</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QLineEdit" name="logdir">
             <property name="text">
              <string>data/output/ti/logs</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Finetune Config</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLineEdit" name="data_root">
             <property name="text">
              <string>data/input/images/out</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QComboBox" name="base"/>
           </item>
           <item row="17" column="0" colspan="2">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="2">
            <widget class="QLineEdit" name="placeholder_tokens">
             <property name="text">
              <string>'*'</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Init Word</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
